crypto/bn/build.info

```diff
@@ -116,11 +116,6 @@ IF[{- !$disabled{'deprecated-3.0'} -}]
 ENDIF
 SOURCE[../../providers/libfips.a]=$COMMON $BNASM
 DEFINE[../../providers/libfips.a]=$BNDEF
-# Because some CPUID implementations use some BN assembler (!!!), we
-# must include assembler code into the legacy provider under the same
-# conditions as CPUID code is included.  See ../build.info
-SOURCE[../../providers/liblegacy.a]=$BNASM
-DEFINE[../../providers/liblegacy.a]=$BNDEF
```

Comment:
So, is the comment that you are deleting here just wrong?

Reply:
Yes, there is no longer anything in CPUID code(s) that would depend on BNASM files.